eap74-rhel8-payg-multivm/src/main/scripts/jbosseap-setup-redhat.sh

```diff
@@ -29,14 +29,15 @@ STORAGE_ACCOUNT_NAME=${16}
 CONTAINER_NAME=${17}
 RESOURCE_GROUP_NAME=${18}
 NUMBER_OF_INSTANCE=${19}
-VM_NAME_PREFIX=${20}
-NUMBER_OF_SERVER_INSTANCE=${21}
-CONFIGURATION_MODE=${22}
-VNET_NEW_OR_EXISTING=${23}
-CONNECT_SATELLITE=${24}
-SATELLITE_ACTIVATION_KEY_BASE64=${25}
-SATELLITE_ORG_NAME_BASE64=${26}
-SATELLITE_VM_FQDN=${27}
+ADMIN_VM_NAME=${20}
+VM_NAME_PREFIX=${21}
+NUMBER_OF_SERVER_INSTANCE=${22}
+CONFIGURATION_MODE=${23}
+VNET_NEW_OR_EXISTING=${24}
+CONNECT_SATELLITE=${25}
+SATELLITE_ACTIVATION_KEY_BASE64=${26}
+SATELLITE_ORG_NAME_BASE64=${27}
+SATELLITE_VM_FQDN=${28}
```

Comment:
This will have to change. See https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1620889 .